graylog2-web-interface/src/views/components/datatable/Headers.tsx

```diff
@@ -32,6 +32,10 @@ const StyledTh = styled.th(({ isNumeric }: { isNumeric: boolean }) => css`
   ${isNumeric ? 'text-align: right' : ''}
 `);
 
+const CentratedTh = styled.th`
```

Comment:
Suggestion for better naming:

```suggestion
const CenteredTh = styled.th`
```